client/src/app/components/keith-test/keith-test/keith-test.component.scss

```diff
@@ -0,0 +1,41 @@
+$backgroundColor: #0d48a13a;
```

Comment:
Nice use of the SCSS variables, they are awesome! Thinking higher level, something like a background color could be extracted out to a global variables file that we can then import into the component scss files and use across the app in order to keep theming consistent and _much_ easier to change later. Also check out the theme file (src>scss folder) for some of the custom theming we have going on.